index.html

```diff
@@ -2884,7 +2981,72 @@ <h3>Binding to Physical Identity</h3>
+    to the school's computer lab. See them to request access".
+    </p><p>
+      From this usage it is ambgious as to whether or not `did:example:abc`
+      refers to a specific teacher or to whomever is the current teacher.
+    </p><p>
+      Only with further statements might we be able to discern the difference.
+    </p><p>
+      But it's still tricky. For example the subject in the following statement
+      ```turtle
+       did:example:abc foaf:name "Bob Smith" . 
+      ```
+     remains ambiguous. If `did:example:abc` refers to a specific human
+      being, then the statement is taken as an attestation about that
+      particular human's name. However, if `did:example:abc` is used to 
+      refer to the _current_ teacher, it is also valid as the current
+      teacher does have that name. In this case the ambiguity is
+      immaterial.
+    </p><p>
+        However, in a statement like
+      ```turtle
+      did:example:abc http://law.example/convicted http://calaw.example/PenalCode647b . 
+      ```     
+    The difference becomes vital. The statement in English could be
+    "the person referred to by did:example:abc has been convicted of
+    California Penal Code 647b." But which person(s) did we mean? Did we
+    mean to say one, some, or all of the teachers of computer science at
+    the school have been convicted of violating Pen647b? Or is it meant to
+    say that a particular individual teacher, perhaps the one named "Bob
+    Smith", has been convicted of said crime? 
+    </p>
+    <p>
+      The challenge is particularly difficult in situations where the
+      subject is fundamentally uninvolved in the issuance of the
+      credential. For example, an identifier might be used by a school to
+      refer to a teacher, and students and or parents may use that
+      identifier to make statements about the teacher, with neither the
+      teacher nor the school involved. In these cases, it is easy to
+      imagine that the subtle nuance of the school's intended meaning, e.g., "any current teacher of the computer science class", gets lost
+      and the identifier misused by parents and
+      students to refer to a specific teacher, quite likely in contexts
+      where neither the school nor the teacher is aware of the
+      conversation.
+    </p>
+    <p>
+      In natural language, these ambiguities are often easily ignored or
+      corrected. In digital media, it's vital that context be evaluated to
+      establish the intended referent, especially when identifiers are
+      used in different contexts by different issuers, e.g.: on a school
+      website by the school, but in a social networking app by parents and
+      students.
+    </p>
+    <p>
+      In short, the context in which identifiers are created and used
+      must be considered when relying on any particular interpretation
+      of the subject of any particular identifier.
+    </p>
 
+
```

Comment:
This is JUST to make this chunk of source HTML follow the same line length and indentation conventions as the rest of the document. There are some suggestions to follow, regarding punctuation and such, which will be made as smaller suggestions.
~~~suggestion
      <h2>Identifier Ambiguity</h2>
        <p>
Even in cases where the subject of the identifier proves control, the
interpretation of the subject remains contextual and potentially ambiguous.
        </p>
        <p>
For example, a school might issue a credential about the teacher of _Intro
to Computer Science_, using `did:example:abc` as a subject identifier,
saying in effect "did:example:abc is the teacher of _Intro to Computer
Science_" and "did:example:abc controls access to the school's computer
lab. See them to request access".
        </p>
        <p>
From this usage it is ambgious as to whether or not `did:example:abc`
refers to a specific teacher or to whomever is the current teacher.
        </p>
        <p>
Only with further statements might we be able to discern the difference.
        </p>
        <p>
But it's still tricky. For example the subject in the following statement
```turtle
did:example:abc foaf:name "Bob Smith" .
```
remains ambiguous. If `did:example:abc` refers to a specific human
being, then the statement is taken as an attestation about that
particular human's name. However, if `did:example:abc` is used to 
refer to the _current_ teacher, it is also valid as the current
teacher does have that name. In this case the ambiguity isimmaterial.
        </p>
        <p>
However, in a statement like
```turtle
did:example:abc http://law.example/convicted http://calaw.example/PenalCode647b . 
```     
The difference becomes vital. The statement in English could be
"the person referred to by did:example:abc has been convicted of
California Penal Code 647b." But which person(s) did we mean? Did we
mean to say one, some, or all of the teachers of computer science at
the school have been convicted of violating Pen647b? Or is it meant to
say that a particular individual teacher, perhaps the one named "Bob
Smith", has been convicted of said crime? 
        </p>
        <p>
The challenge is particularly difficult in situations where the
subject is fundamentally uninvolved in the issuance of the
credential. For example, an identifier might be used by a school to
refer to a teacher, and students and or parents may use that
identifier to make statements about the teacher, with neither the
teacher nor the school involved. In these cases, it is easy to
imagine that the subtle nuance of the school's intended meaning,
e.g., "any current teacher of the computer science class", gets lost
and the identifier misused by parents and
students to refer to a specific teacher, quite likely in contexts
where neither the school nor the teacher is aware of the
conversation.
        </p>
        <p>
In natural language, these ambiguities are often easily ignored or
corrected. In digital media, it's vital that context be evaluated to
establish the intended referent, especially when identifiers are
used in different contexts by different issuers, e.g.: on a school
website by the school, but in a social networking app by parents and
students.
        </p>
        <p>
In short, the context in which identifiers are created and used
must be considered when relying on any particular interpretation
of the subject of any particular identifier.
        </p>
~~~